composer.json

```diff
@@ -28,7 +28,7 @@
         }
```

Comment:
You will need to remove the "replace" section of this file; it's no longer a drop-in replacement for the ZF pacakges after the changes you're proposing.

Reply:
Time to drop the bridge aswell as its a new major?